src/Symfony/Component/Workflow/Exception/NotEnabledTransitionException.php

```diff
@@ -28,7 +28,7 @@ public function __construct(
         private TransitionBlockerList $transitionBlockerList,
         array $context = [],
     ) {
-        parent::__construct($subject, $transitionName, $workflow, \sprintf('Transition "%s" is not enabled for workflow "%s".', $transitionName, $workflow->getName()), $context);
+        parent::__construct($subject, $transitionName, $workflow, \sprintf('Can not apply transition to "%s" for workflow "%s".', $transitionName, $workflow->getName()), $context);
```

Comment:
the first parameter is the name of the transition, so the sentence is confusing:
`Can not apply transition to "transitionName" ...?`

Or am I missing sth.?

Reply:
So changer to can not apply transition of %s ?